### DISCUSSION TITLE
Using :from with nav:products

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#6088] [General]
Hi

How come this works

```
{{ nav :from="segment_1" }} ... {{ /nav }}
```

But this doesn't?

```
{{ nav:products :from="segment_1" }} ... {{ /nav }}
```

Could this be added to a future update?

**Comment by edalzell** [comment]
Why would you want that, what's your use case?

**Reply by stephenmeehanuk** [reply]
I've since decided to use an ordered collection instead, and by using `:from="url"` I was able to achieve what I needed. Which was to display a list of entires based on where the user is in the navigation tree.

I would have preferred to use taxonomies to give entries a structure as it allows for an entry to be associated with more than one taxonomy term. But I really needed the ability to nest entries, something that taxonomies currently don't support, which is a shame.

**Comment by ryanmitchell** [comment]
Heres a gist for a custom tag I use for this sort of thing:
https://gist.github.com/ryanmitchell/c385bbb0b505cd80fe88aa86af036b9c

Your usage may vary, so be prepared to change the code.

**Reply by stephenmeehanuk** [reply]
Thanks, really appreciate sharing some code, but I've decided to approach this from another way for now.

**Comment by jasonvarga** [comment]
The `from` parameter is only supported when dealing with collections.

When you do `{{ nav }}`, it is technically an alias for `{{ nav:collection:pages }}` which targets the `pages` collection.

When you do `{{ nav:products }}`, you're targeting a nav named products.

**Reply by stephenmeehanuk** [reply]
Cheers, yes I was trying to use `{{ nav:products }}` in the same way as a structured collection `{{ nav:collection:products }}`. I've just decided to use a structured collection for now, as a workaround for no nested taxonomy support.

Are nested taxonomy terms on the horizon? I know it's a difficult thing to implement, but for this a project I'm working on it'd be a great feature to have.
